### ISSUE TITLE
IOError on ubuntu server

### ISSUE THREAD

**Issue by thezawad** [thezawad/teletubemp3#1]
```
root@hostname:~/teletubemp3# python ytgr.py
[+] Server is Listenining [+]
[=] Type Command from Telegram [=]
Luis Fonsi - Despacito ft. Daddy Yankee
https://www.youtube.com/watch?v=kJQP7kiw5Fk
WARNING: Assuming --restrict-filenames since file system encoding cannot encode all characters. Set the LC_ALL environment variable to fix this.
[youtube] kJQP7kiw5Fk: Downloading webpage
[youtube] kJQP7kiw5Fk: Downloading video info webpage
[youtube] kJQP7kiw5Fk: Extracting video information
[youtube] kJQP7kiw5Fk: Downloading MPD manifest
[download] Destination: Luis_Fonsi_-_Despacito_ft._Daddy_Yankee-kJQP7kiw5Fk.webm
[download] 100% of 4.49MiB in 00:00
[ffmpeg] Destination: Luis_Fonsi_-_Despacito_ft._Daddy_Yankee-kJQP7kiw5Fk.mp3
WARNING: Your copy of avconv is outdated, update avconv to version 10-0 or newer if you encounter any errors.
Deleting original file Luis_Fonsi_-_Despacito_ft._Daddy_Yankee-kJQP7kiw5Fk.webm (pass -k to keep)
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/telepot/__init__.py", line 989, in collector
    callback(item)
  File "ytgr.py", line 46, in handle
    bot.sendAudio(chat_id,audio=open(title+"-"+watchid+".mp3",'rb'))
IOError: [Errno 2] No such file or directory: u'Luis Fonsi - Despacito ft. Daddy Yankee-kJQP7kiw5Fk.mp3'
```
This error occurs while bot tries to send file.

**Comment by thezawad** [comment]
Issue fixed in commit 9c13d40

**Comment by thezawad** [comment]
```
[+] Server is Listenining [+]
[=] Type Command from Telegram [=]
Command from client : yt monermanush 
'Moner Manush' - Anupam Roy Feat. Satyaki Banerjee & Babul Supriyo - Coke Studio@MTV Season 4
https://www.youtube.com/watch?v=o5iNbmDbViA
[youtube] o5iNbmDbViA: Downloading webpage
[youtube] o5iNbmDbViA: Downloading video info webpage
[youtube] o5iNbmDbViA: Extracting video information
[youtube] o5iNbmDbViA: Downloading MPD manifest
[download] Destination: Moner_Manush_-_Anupam_Roy_Feat._Satyaki_Banerjee_Babul_Supriyo_-_Coke_Studio@MTV_Season_4-o5iNbmDbViA.webm
[download] 100% of 9.76MiB in 00:0005MiB/s ETA 00:00known ETA
[ffmpeg] Destination: Moner_Manush_-_Anupam_Roy_Feat._Satyaki_Banerjee_Babul_Supriyo_-_Coke_Studio@MTV_Season_4-o5iNbmDbViA.mp3
WARNING: Your copy of avconv is outdated, update avconv to version 10-0 or newer if you encounter any errors.
Deleting original file Moner_Manush_-_Anupam_Roy_Feat._Satyaki_Banerjee_Babul_Supriyo_-_Coke_Studio@MTV_Season_4-o5iNbmDbViA.webm (pass -k to keep)
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/telepot/__init__.py", line 989, in collector
    callback(item)
  File "ytgr.py", line 48, in handle
    bot.sendAudio(chat_id,audio=open(filename,'rb'))
IOError: [Errno 2] No such file or directory: u"'Moner_Manush'_-_Anupam_Roy_Feat._Satyaki_Banerjee_&_Babul_Supriyo_-_Coke_Studio@MTV_Season_4-o5iNbmDbViA.mp3"
```

**Comment by thezawad** [comment]
another IOError issue.

**Comment by thezawad** [comment]
```
[+] Server is Listenining [+]
[=] Type Command from Telegram [=]
Command from client : yt monermanush 
'Moner Manush' - Anupam Roy Feat. Satyaki Banerjee & Babul Supriyo - Coke Studio@MTV Season 4
https://www.youtube.com/watch?v=o5iNbmDbViA
WARNING: Assuming --restrict-filenames since file system encoding cannot encode all characters. Set the LC_ALL environment variable to fix this.
[youtube] o5iNbmDbViA: Downloading webpage
[youtube] o5iNbmDbViA: Downloading video info webpage
[youtube] o5iNbmDbViA: Extracting video information
[youtube] o5iNbmDbViA: Downloading MPD manifest
[download] Destination: Moner_Manush_-_Anupam_Roy_Feat._Satyaki_Banerjee_Babul_Supriyo_-_Coke_Studio@MTV_Season_4-o5iNbmDbViA.webm
[download] 100% of 9.76MiB in 00:00
[ffmpeg] Destination: Moner_Manush_-_Anupam_Roy_Feat._Satyaki_Banerjee_Babul_Supriyo_-_Coke_Studio@MTV_Season_4-o5iNbmDbViA.mp3
WARNING: Your copy of avconv is outdated, update avconv to version 10-0 or newer if you encounter any errors.
Deleting original file Moner_Manush_-_Anupam_Roy_Feat._Satyaki_Banerjee_Babul_Supriyo_-_Coke_Studio@MTV_Season_4-o5iNbmDbViA.webm (pass -k to keep)
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/telepot/__init__.py", line 989, in collector
    callback(item)
  File "ytgr.py", line 49, in handle
    bot.sendAudio(chat_id,audio=open(filename,'rb'))
IOError: [Errno 2] No such file or directory: u'Moner_Manush_-_Anupam_Roy_Feat._Satyaki_Banerjee_&_Babul_Supriyo_-_Coke_Studio@MTV_Season_4-o5iNbmDbViA.mp3'
```

IOError for ` ' , & `

**Comment by thezawad** [comment]
Fixed with 975289d

**Comment by thezawad** [comment]
08282d1  another commit
fixes ()[]{}
